test/consensus/sumeragi_test.cpp

```diff
@@ -26,16 +26,34 @@ limitations under the License.
 #include <memory>
 #include <thread>
 
-#include "../../core/service/json_parse_with_json_nlohmann.hpp"
+#include "../../core/service/json_parse_with_json_nlohman.hpp"
 
 #include "../../core/service/peer_service.hpp"
 #include "../../core/crypto/hash.hpp"
 
+template<typename T>
+using Transaction = transaction::Transaction<T>;
+template<typename T>
+using ConsensusEvent = event::ConsensusEvent<T>;
+template<typename T>
+using Add = command::Add<T>;
+template<typename T>
+using Transfer = command::Transfer<T>;
+
+void setAwkTimer(int const sleepMillisecs, std::function<void(void)> const action) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'action' should be passed by reference.](https://www.codacy.com/app/soramitsu/iroha/file/4478128097/issues/source?bid=3712491&fileBranchId=3923596#l43)